webapp/packages/core-ui/src/InlineEditor/InlineEditor.tsx

```diff
@@ -31,6 +33,7 @@ export interface InlineEditorProps extends Omit<React.InputHTMLAttributes<HTMLIn
   active?: boolean;
   loading?: boolean;
   style?: ComponentStyle;
+  hintsState?: HintsState;
```

Comment:
can't be part of this component, you need to track keys and value, you can track it outside of this component